### DISCUSSION TITLE
Landmarks for guiding segmentation

### DISCUSSION THREAD

**Discussion by che85** [Project-MONAI/MONAILabel#1558] [Q&A]
Hello, 

other than only training a semantic segmentation by a single input image, I would want to provide fixed anatomical landmarks to guide segmentation. 

Is that possible? 

Thanks in advance for your help.

Christian

**Comment by diazandr3s** [comment]
Hi @che85,

This is a good question. Yes, this is possible. In fact, this is the mechanism we use to train the DeepEdit model. 
For the DeepEdit model, we simulate clicks and use them to guide training/inference. These landmarks are concatenated to the input tensor and considered as an extra input channel.

Here are the transforms we use for that in DeepEdit: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/trainers/deepedit.py#L115-L118

Hope this helps,
